grammar.js

```diff
@@ -1,3 +1,10 @@
+const commaSep1 = (term) => seq(
+  term,
+  repeat(seq(',', term))
+);
+
+const identifierRegex = /[a-z_][A-Za-z0-9_]*/
```

Comment:
This may not be perfect here, but it's a start, and it was needed for the `namedTupleLiteral` rule.

Reply:
Can you move/reuse the one I did for symbols? I think that fully does the full range of identifiers, though it's very ugly.

Reply:
Sure, will do. I'll make that change and then "squash" it into this commit.

Reply:
....and done!